ios/Exponent.xcodeproj/project.pbxproj

```diff
@@ -1172,7 +1173,7 @@
 		FC756CCA24CB542800665AE6 /* EXScopedNotificationSerializer.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; path = EXScopedNotificationSerializer.h; sourceTree = "<group>"; };
 		FCA11CA524C1215B00C1FC54 /* EXScopedNotificationCategoriesModule.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = EXScopedNotificationCategoriesModule.m; sourceTree = "<group>"; };
 		FCA11CA624C1215B00C1FC54 /* EXScopedNotificationCategoriesModule.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = EXScopedNotificationCategoriesModule.h; sourceTree = "<group>"; };
-		FE89956BF2504EB0A76B89AD /* MutableValueSetterProxy.h */ = {isa = PBXFileReference; explicitFileType = undefined; fileEncoding = 4; includeInIndex = 0; lastKnownFileType = sourcecode.c.h; name = MutableValueSetterProxy.h; path = MutableValueSetterProxy.h; sourceTree = "<group>"; };
+		FE89956BF2504EB0A76B89AD /* MutableValueSetterProxy.h */ = {isa = PBXFileReference; explicitFileType = undefined; fileEncoding = 4; includeInIndex = 0; lastKnownFileType = sourcecode.c.h; path = MutableValueSetterProxy.h; sourceTree = "<group>"; };
```

Comment:
What version of Xcode are you using? These look like rather safe changes, but if we don't need to introduce them if they eg. come from an old Xcode, let's not do it! 👍

Reply:
Xcode 12.0, maybe the current project is built on 11?

Reply:
I'm not sure it matters on which Xcode the project is built, what I meant is that Xcode did not propose to modify this file when I opened it on my computer and I was checking if something may be off, but if your installation is up-to-date then this is not a problem! — ✅

Reply:
ah okay I see what you mean now, well I reverted the changes anyways so as long as that passes I'll merge it without these pbxproj changes included 👍